### DISCUSSION TITLE
lm或者newcm的\mathcal字体为什么那么奇怪？

### DISCUSSION THREAD

**Discussion by yuan-liu16** [tuna/thuthesis#912] [Q&A] [answered]
我就是个小白。想知道Thuthesis里面，选择了lm或者newcm的字体设置，为什么\mathcal字体看起来和LaTex里的完全不一样(RevTex模板里的那种)？明明普通字母看上去差不多。大神可以帮忙想个办法让它一样吗
我想要的如图所示[
![示意](https://github.com/tuna/thuthesis/assets/153643108/e82e10e9-dd0d-44d4-ab6f-9b601f67c0c4)
](url)

**Comment by zepinglee** [comment, marked as answer]
revtex 的字体是 `amsmath` 提供的。模板中为了适配各种各样的需求所以采用了 `unicode-math` + OpenType Math 字体方案，这些字体的 `\mathcsl` 字形设计成了这样。

revtex:
<img width="101" alt="Screenshot 2024-01-31 at 19 04 37" src="https://github.com/tuna/thuthesis/assets/12290822/565b5f46-f0c0-4e23-aa09-23e3559c2dc6">

Latin Modern:
<img width="79" alt="Screenshot 2024-01-31 at 19 05 32" src="https://github.com/tuna/thuthesis/assets/12290822/03a1fdf6-609d-4c74-9a43-af02716e63fb">

New Computer Modern:
<img width="80" alt="Screenshot 2024-01-31 at 19 06 17" src="https://github.com/tuna/thuthesis/assets/12290822/ea46085d-70c0-4bb6-ae5c-2c21072d97bc">

一种办法是设置 `math-font  = newtx`，并注释掉这行（记得更新 `.cls` 文件）。

https://github.com/tuna/thuthesis/blob/fedf6da50f2e6f3242488c952f789eea104e5d6d/thuthesis.dtx#L3055

但不排除还有别的问题，你可以自行研究一下。

**Comment by yuan-liu16** [comment]
这个是可以的。然后发现想用XITS模仿PRL的字体的话，可以加上
\let\svmathcal\mathcal
\DeclareSymbolFontAlphabet{\mathcal}   {symbols}
\DeclareMathAlphabet{\mathcal}{OMS}{cmsy}{m}{n}
\SetMathAlphabet{\mathcal}{bold}{OMS}{cmsy}{b}{n}
